state-recover/test-cases/src/integrationTest/kotlin/linea/staterecover/StateRecoverAppIntTest.kt

```diff
@@ -1,197 +0,0 @@
-package linea.staterecover
-
-import build.linea.clients.StateManagerClientV1
-import build.linea.clients.StateManagerV1JsonRpcClient
-import build.linea.contract.l1.LineaContractVersion
-import build.linea.contract.l1.LineaRollupSmartContractClientReadOnly
-import build.linea.contract.l1.Web3JLineaRollupSmartContractClientReadOnly
-import build.linea.staterecover.clients.el.ExecutionLayerJsonRpcClient
-import io.micrometer.core.instrument.simple.SimpleMeterRegistry
-import io.vertx.core.Vertx
-import io.vertx.junit5.VertxExtension
-import linea.build.staterecover.clients.VertxTransactionDetailsClient
-import linea.domain.RetryConfig
-import linea.log4j.configureLoggers
-import linea.staterecover.clients.blobscan.BlobScanClient
-import linea.web3j.Web3JLogsSearcher
-import net.consensys.linea.BlockParameter
-import net.consensys.linea.jsonrpc.client.RequestRetryConfig
-import net.consensys.linea.jsonrpc.client.VertxHttpJsonRpcClientFactory
-import net.consensys.linea.metrics.micrometer.MicrometerMetricsFacade
-import net.consensys.linea.testing.submission.AggregationAndBlobs
-import net.consensys.linea.testing.submission.loadBlobsAndAggregationsSortedAndGrouped
-import net.consensys.linea.testing.submission.submitBlobsAndAggregationsAndWaitExecution
-import net.consensys.zkevm.coordinator.clients.smartcontract.LineaRollupSmartContractClient
-import net.consensys.zkevm.ethereum.ContractsManager
-import net.consensys.zkevm.ethereum.Web3jClientManager
-import org.apache.logging.log4j.Level
-import org.apache.logging.log4j.LogManager
-import org.assertj.core.api.Assertions.assertThat
-import org.awaitility.Awaitility.await
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
-import java.net.URI
-import kotlin.time.Duration.Companion.milliseconds
-import kotlin.time.Duration.Companion.minutes
-import kotlin.time.Duration.Companion.seconds
-import kotlin.time.toJavaDuration
-
-@ExtendWith(VertxExtension::class)
-class StateRecoverAppIntTest {
```

Comment:
Did I understand it correctly, that this test is now a part of `StateRecoveryManualReplayToLocalStackIntTest`?